cryptotik/binance.py

```diff
@@ -126,7 +126,8 @@ def private_api(self, url, params={}, http_method='GET'):
         query += "&signature={}".format(signature)
         result = requests.request(http_method,
                                 url + "?" + query,
-                                headers={"X-MBX-APIKEY": self.apikey})
+                                headers={"X-MBX-APIKEY": self.apikey},
+                                timeout=3)
```

Comment:
Timeout should be inherited from the init method: https://github.com/peerchemist/cryptotik/blob/master/cryptotik/bittrex.py#L22